redash/authentication/remote_user_auth.py

```diff
@@ -12,9 +13,11 @@
 def login():
     next_path = request.args.get('next')
 
+    index_url_with_next = url_for('redash.index', next=next_path)
+
```

Comment:
In general I'm nervous about code cleanups like this because it's extra surface area for conflicts when integrating with upstream

Reply:
Alright, will back it out.